docs/en/quick-answers/developing/create-publishes-via-api.md

```diff
@@ -0,0 +1,88 @@
+This ultimately uses the core sgtk API method to register the PublishedFile, but it also provides a framework around collection, validation and publishing, that can be customized.
+As well as the publish API documentation, we have examples of writing your own publish plugins in our [pipeline turorial](https://developer.shotgunsoftware.com/cb8926fc/?title=Pipeline+Tutorial).
+
+## Using the register_publish() API method
+While it is possible to create publish records in Shotgun using a raw shotgun API call, we would strongly recommend using Toolkit's convenience methods.
+All toolkit apps that create publishes are using a API utility method method called [`sgtk.util.register_publish()`](https://developer.shotgunsoftware.com/tk-core/utils.html#sgtk.util.register_publish).
+
+Basically, this method will create a new  PublishedFile entity in Shotgun and is trying to make that easy by using toolkit concepts. Your code would need to do something along these lines:
+
+```python
+# Get access to the toolkit API
+import sgtk
+
+# this is the file we want to publish.
+file_to_publish = "/mnt/projects/proj/seq_abc/shot_123/comp/foreground.v034.nk"
+
+# alternatively, for file sequences, we can just use
+# a standard sequence token
+# file_to_publish = "/mnt/projects/proj/seq_abc/shot_123/comp/renders/v034/foreground.%04d.exr"
+
+# The name for the publish should be the filename
+# without any version number or extension
+name = "foreground"
+
+# initialize an API object. If you have used the toolkit folder creation 
+# to create the folders where the publish file resides, you can use this path
+# to construct the API object. Alternatively you can create it from any Shotgun
+# entity using the sgtk_from_entity() method.
+tk = sgtk.sgtk_from_path(file_to_publish)
+
+# use the file to extract the context. The context denotes the current work area in toolkit
+# and will control which entity and task the publish will be linked up to. If you have used the toolkit
+# folder creation to create the folders where the publish file resides, you can use this path
+# to construct the context.
+ctx = tk.context_from_path(file_to_publish)
+
+# alternatively, if the file you are trying to publish is not in a location that is
+# recognized by toolkit, you could create a context directly from a Shotgun entity instead:
+ctx = tk.context_from_entity("Shot", 123)
+ctx = tk.context_from_entity("Task", 123)
+
+# Finally, run the publish command.
+# the third parameter (file.nk) is typically the file name, without a version number.
+# this makes grouping inside of Shotgun easy. The last parameter is the version number.
+sgtk.util.register_publish(
+  tk, 
+  ctx, 
+  file_to_publish, 
+  name, 
+  published_file_type="Nuke Script",
+  version_number=34
+)
+```
+
+Note that there are also several additional options you can populate in addition to the basic ones shown above:
+
```

Comment:
haha! brilliant. Just want to really drive home that there are some **additional ones**
I think this whole doc feels odd. I cut out a lot of the extra stuff at the top and made it purely about the API thing. I plan to rewrite it so that it covers path resolving and publishing then it will feel more distinctive from the reference docs, and I'll remove the parameters section entirely.